### PR TITLE
ci(postman): Modify confirm call to use client-secret instead of relying on API calls

### PR DESCRIPTION
**Pull request by pixincreate**
## Type of Change

- [x] Refactoring
- [x] CI/CD

## Description

Currently, confirm calls in postman relies only on API keys and it works as expected. We're supposed to follow the SDK flow where the confirm calls rely on `client_secret` and `publishable_keys`. So this change.

## Motivation and Context

We get client_secret after Payment create has been made, and use that client secret in the confirm abiding by SDK flows.
This was before:
```curl
curl --location 'https://sandbox.hyperswitch.io/payments/<payment_id>/confirm' \
--header 'Content-Type: application/json' \
--header 'Accept: application/json' \
--header 'api-key: <api-key>' \
--data '{}'
```
This is after:
```curl
curl --location 'https://sandbox.hyperswitch.io/payments/<payment_id>/confirm' \
--header 'Content-Type: application/json' \
--header 'Accept: application/json' \
--header 'api-key: <publishablekey>' \
--data '{
    "client_secret": "<client-secret-obtained-in-payment-create-response>"
}'
```
This change has been made to all collections.

## How did you test it?

Adyen_UK:
image

Bluesnap:
image

Checkout:
image

Payme:
image

PayPal:
image

Stripe:
image

Trustpay:
image

## Checklist

- [x] I formatted the code `cargo +nightly fmt --all`
- [x] I reviewed the submitted code
- [x] I added unit tests for my changes where possible

### CODE REVIEW

**Review by knutties — DISMISSED** [review]

File: postman/collection-dir/stripe/Flow Testcases/Happy Cases/Scenario2-Create payment with confirm false/Payments - Confirm/event.test.js [hunk 1]
```diff
@@ -43,16 +43,4 @@ if (jsonData?.client_secret) {
    console.log("- use {{client_secret}} as collection variable for value",jsonData.client_secret);
 } else {
    console.log('INFO - Unable to assign variable {{client_secret}}, as jsonData.client_secret is undefined.');
-};
-
-// Response body should have value "succeeded" for "status"
```
Comment (SanchithHegde) [inline postman/collection-dir/stripe/Flow Testcases/Happy Cases/Scenario2-Create payment with confirm false/Payments - Confirm/event.test.js:48]: Why were these asserts removed?

File: postman/collection-dir/stripe/Flow Testcases/Happy Cases/Scenario3-Create payment without PMD/Payments - Confirm/event.test.js [hunk 2]
```diff
@@ -43,10 +43,4 @@ if (jsonData?.client_secret) {
    console.log("- use {{client_secret}} as collection variable for value",jsonData.client_secret);
 } else {
    console.log('INFO - Unable to assign variable {{client_secret}}, as jsonData.client_secret is undefined.');
-};
-
-// Response body should have value "succeeded" for "status"
-if (jsonData?.status) {
-pm.test("[POST]::/payments:id/confirm - Content check if value for 'status' matches 'succeeded'", function() {
```
Comment (SanchithHegde) [inline postman/collection-dir/stripe/Flow Testcases/Happy Cases/Scenario3-Create payment without PMD/Payments - Confirm/event.test.js:50]: Again, why was this assert removed?

File: postman/collection-dir/stripe/Flow Testcases/Happy Cases/Scenario7-Create 3DS payment with confrm false/Payments - Confirm/event.test.js [hunk 3]
```diff
@@ -43,15 +43,4 @@ if (jsonData?.client_secret) {
    console.log("- use {{client_secret}} as collection variable for value",jsonData.client_secret);
 } else {
    console.log('INFO - Unable to assign variable {{client_secret}}, as jsonData.client_secret is undefined.');
-};
-
-// Response body should have value "requires_customer_action" for "status"
```
Comment (SanchithHegde) [inline postman/collection-dir/stripe/Flow Testcases/Happy Cases/Scenario7-Create 3DS payment with confrm false/Payments - Confirm/event.test.js:48]: Same here

File: postman/collection-dir/stripe/Payments/Session Token/event.test.js [hunk 4]
```diff
@@ -9,7 +9,7 @@ pm.test("[POST]::/payments/session_tokens - Content-Type is application/json", f
 });
 
 const responseJson = pm.response.json();
-
+console.log(responseJson)
```
Comment (SanchithHegde) [inline postman/collection-dir/stripe/Payments/Session Token/event.test.js:12]: Do we absolutely need this?

**Review by SanchithHegde — COMMENTED** [review]
Also, please run the `postman/collection-dir` directory through a formatter, say `prettier`. The formatting was slightly disturbed in this PR.

We can take up the re-formatting of files in a separate PR, not a problem.

File: postman/collection-dir/stripe/Payments/Session Token/event.test.js (hunk 4 above)
Comment (pixincreate) [reply postman/collection-dir/stripe/Payments/Session Token/event.test.js:12]: Oh, it was added for debugging ApplePay related PR, will remove that

File: postman/collection-dir/stripe/Flow Testcases/Happy Cases/Scenario2-Create payment with confirm false/Payments - Confirm/event.test.js (hunk 1 above)
Comment (pixincreate) [reply postman/collection-dir/stripe/Flow Testcases/Happy Cases/Scenario2-Create payment with confirm false/Payments - Confirm/event.test.js:48]: Unsure how these were removed. I only touched the requests.json for confirm calls. Added back.

**Comment by pixincreate** [comment]
> Also, please run the `postman/collection-dir` directory through a formatter, say `prettier`. The formatting was slightly disturbed in this PR.
> 
> We can take up the re-formatting of files in a separate PR, not a problem.

Ran the below command and it modified 4k+ files.
```sh
npm install -g prettier 
prettier --write .
```

**Review by SanchithHegde — APPROVED** [review]

**Review by Gnanasundari24 — APPROVED** [review]

**Pull request merged**